e2e/firstTest.spec.js

```diff
@@ -0,0 +1,240 @@
+require('coffeescript').register();
```

Comment:
Is the coffeescript necessary for the utility libraries out of heap/heap?

Reply:
Yep.  The utilities are in coffeescript, and detox doesn't recognize coffeescript files without `coffeescript/register`